tasks.py

```diff
@@ -15,6 +16,7 @@
 from pathlib import Path
 from typing import Optional
 
+import requests
```

Comment:
Is requests installed?

Reply:
I think so, on the basis of the `setup-cached-python` step that runs before this. It worked when I copy-pasted that action source into my test repo. That said, it looks like requests is just a transitive dependency and not in on `requirements` file. Let me know if you think we need to add it there or if my understanding of the setup flow is otherwise off.

Reply:
That said I'm not totally sure how to interpret the failing mypy step, which pertains to `requests`...

Reply:
Added `requests` and `types-requests` to the dev requirements.